packages/expo-image/src/web/style.tsx

```diff
@@ -0,0 +1,121 @@
+const style = `
+        .cross-dissolve {
```

Comment:
Kinda weird indentation 😅 I would say no indent for selectors is the best choice